src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2091,7 +2105,9 @@ export default class Aircraft {
             if (this.isLanded()) {
                 this.target.speed = 0;
             }
-        } else if (this.fms.currentWaypoint().navmode === WAYPOINT_NAV_MODE.FIX) {
+        }
+
+        else if (this.fms.currentWaypoint().navmode === WAYPOINT_NAV_MODE.FIX) {
```

Comment:
make sure an else if is on the same line as the closing brace for the previous conditional.